sjsonnet/src/sjsonnet/Std.scala

```diff
@@ -549,6 +549,7 @@ class Std {
         q.removeFirst().force match {
           case v: Val.Arr => v.asLazyArray.reverseIterator.foreach(q.push)
           case s: Val.Str => out.write(s.value)
+          case _ =>
```

Comment:
For compatibility with jsonnet behavior, I think this should throw an error rather than silently ignoring elements of unexpected types:

https://github.com/google/jsonnet/blob/913281d203578bb394995bacc792f2576371e06c/stdlib/std.jsonnet#L321-L327

Maybe we can change this in a followup / separate PR?